x-pack/plugins/cases/server/client/attachments/validators.test.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { validateFindCommentsPagination } from './validators';
+
+const ERROR_MSG =
+  'The number of documents is too high. Paginating through more than 10,000 documents is not possible.';
+
+describe('validators', () => {
+  describe('validateFindCommentsPagination', () => {
+    it('returns if page is undefined', () => {
```

Comment:
nit: I think you mean it does not throw if `perPage` is defined and `page` is not, right?

Reply:
You're right 👍